packages/jss-plugin-syntax-rule-value-function/package.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "jss-plugin-syntax-rule-value-function",
+  "description": "JSS plugin for function syntax",
+  "version": "0.1.0",
+  "author": {
+    "name": "Oleg Slobodskoi",
+    "email": "oleg008@gmail.com"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git@github.com:cssinjs/jss.git"
+  },
+  "keywords": [
+    "jss",
+    "style",
+    "sheet",
+    "stylesheet",
+    "css",
+    "components",
+    "composable",
+    "css in js",
+    "css-in-js",
+    "observable",
```

Comment:
these last 3 are from observable, I think we can use: "dynamic", "function", "reactive", "animation"